### DISCUSSION TITLE
Bus error dopo flash di HassOS su Mini PC

### DISCUSSION THREAD

**Discussion by ermestre** [indomus/forum#2208] [HUB: Home Assistant]
Tutto bene fino all’ultimo comando che non riesco proprio a farlo ossia
sudo shutdown now 
Che mi risponde 
Bus error (
![image](https://user-images.githubusercontent.com/80640861/209411080-629df23d-a42e-4146-8d22-9c03e4451de9.jpg)
core dumped)

**Comment by KC-inDomus** [comment]
lascia perdere, spegni e prosegui con la guida
